creusot-contracts/src/std/iter/range.rs

```diff
@@ -50,6 +50,7 @@ pub fn range_inclusive_len<Idx: DeepModel<DeepModelTy = Int>>(r: RangeInclusive<
     }
 }
 
+#[cfg(creusot)]
 impl<Idx: DeepModel<DeepModelTy = Int> + Step> Iterator for RangeInclusive<Idx> {
```

Comment:
This is where `Step` is used. This PR currently just removes these `impl`s. Is that okay or is there a possible replacement?

Reply:
Yes you can remove them since that's purely for specifications.

Reply:
Yeah, the trait and the implementations should remain in the non-creusot config.

Isn't there a way to force Rustc to accept unstable features in a crate used by a stable Rust compiler? That would be unfair, Rust's stdlib is using plenty of unstable features to target stable clients, but we couldn't!

Reply:
As far as I can tell, rustc provides special attributes for std to mark APIs as stable or unstable, and to allow itself to use its own unstable APIs in stable code. That's how it's able to expose an `impl` that depends on `Step` (or `Allocator` or some other unstable trait). Those attributes can't be used outside of std.